src/main/kotlin/uk/gov/dluhc/printapi/config/CorrelationIdMdcConfiguration.kt

```diff
@@ -111,4 +111,4 @@ private fun generateCorrelationId(): String =
     UUID.randomUUID().toString().replace("-", "")
 
 private fun getCurrentCorrelationId(): String =
-    MDC.get(CORRELATION_ID_HEADER) ?: generateCorrelationId()
+    MDC.get(CORRELATION_ID) ?: generateCorrelationId()
```

Comment:
This is the fix! Tiny stupid typo! 😁 
(If I'd have written tests in the first place it would have caught it! 🙄 )